compiler/closureiters.nim

```diff
@@ -864,6 +876,13 @@ proc transformReturnsInTry(ctx: var Ctx, n: PNode): PNode =
 
   of nkSkip:
     discard
+  of nkTryStmt:
+    if n.hasYields:
+      # the inner try will handle theses transformations
```

Comment:
Small nit:

```suggestion
      # the inner try will handle these transformations
```

As a side note: thank you for including these comments, they will really help with understanding this code base into the future :)